### PR TITLE
Lint TensorFlow: Mixed Precision guide

### PR DESCRIPTION
**Pull request by 8bitmp3**
Splitting PR #1736 into smaller digestible bits. Cheers @lamberta @MarkDaoust 

There's a section on a technique for mixed precision training called "loss scaling" (I also read about it here https://arxiv.org/abs/1910.12385). It looks like we can say "a loss scale value" or "a loss scale hyperparameter". Here's my new suggestion:

```diff
The basic concept of loss scaling is simple:
- simply multiply the loss by some large number, say $1024$, 
+ multiply the loss by some large number, say $1024$, 
- We call this number the *loss scale*...
- You can call this number the *loss scale*.
+ and you get the *loss scale* value.
...
```

(I've removed "simply" to avoid repetition/duplication.)

Thank you 🙏

### CODE REVIEW

**Comment by 8bitmp3** [comment]
PTAL @lamberta @MarkDaoust when you got time, cheers 👍

**Comment by lamberta** [comment]
Merge conflict, please resolve

**Comment by 8bitmp3** [comment]
Merge conflict addressed (following the notebook update on Jan 22 by reedwm). PTAL @lamberta, thank you very much.

I ran nbfmt (`python3 -m tensorflow_docs.tools.nbfmt mixed_precision.ipynb`) twice but it doesn't seem to help with the notebook format text 🤔

**Review by lamberta — APPROVED** [review]

**Review by lamberta — DISMISSED** [review]
Should be good now. Make sure to rebase with the master branch when resolving conflicts. And may require a force push to your branch afterwards

**Comment by 8bitmp3** [comment]
@googlebot I consent.

**Comment by MarkDaoust** [comment]
> I consent.

The bot is mad about all the other authors on the extra commits. 

I think you just need to `git rebase master` to fix this.

Or: https://xkcd.com/1597/

**Review by lamberta — APPROVED** [review]
